packages/fxa-auth-server/lib/senders/emails/templates/cadReminderSecond/index.stories.ts

```diff
@@ -0,0 +1,57 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Story, Meta } from '@storybook/html';
+import storybookEmail, { StorybookEmailArgs } from '../../storybook-email';
+
+export default {
+  title: 'Emails/cadReminderSecond',
+} as Meta;
+
+const Template: Story<StorybookEmailArgs> = (args) => storybookEmail(args);
+
+const defaultVariables = {
+  action: 'Sync another device',
+  oneClickLink: true,
+  link: 'http://localhost:3030/connect_another_device?utm_medium=email&utm_campaign=fx-cad-reminder-second&utm_content=fx-connect-device',
+  iosUrl:
+    'https://accounts-static.cdn.mozilla.net/product-icons/apple-app-store.png',
+  androidUrl:
+    'https://accounts-static.cdn.mozilla.net/product-icons/google-play.png',
+  onDesktopOrTabletDevice: true,
+  subject: 'Final Reminder: Complete Sync Setup',
+  privacyUrl:
+    'https://www.mozilla.org/privacy?utm_medium=email&utm_campaign=fx-cad-reminder-second&utm_content=fx-privacy',
+  supportUrl:
+    'https://support.mozilla.org/kb/im-having-problems-with-my-firefox-account?utm_medium=email&utm_campaign=fx-cad-reminder-second&utm_content=fx-support',
+  headerText: 'Last reminder to sync devices!',
+  bodyText:
+    'Syncing another device with Firefox privately keeps your bookmarks, passwords and other Firefox data the same everywhere you use Firefox.',
+  acceptLanguage: 'en-US',
+};
+
+const commonPropsWithOverrides = (
+  overrides: Partial<typeof defaultVariables> = {}
+) =>
+  Object.assign({
+    template: 'cadReminderSecond',
+    layout: 'fxa',
```

Comment:
Does it look like we're going to use `layout: 'fxa'` for all or nearly all of our emails, and any other common props (`acceptLanguage: 'en-US'`)? We can probably make something shared to pull those kinds of props in.

This is more of an aside and not something to do here but I wonder if there's a way to share some of the `defaultVariables` strings with the English Fluent files so there's less copy-and-paste (and then if some copy changes, we won't have to change it in two spots). 🤔